locales/be-BY/src/vision.ts

```diff
@@ -0,0 +1,69 @@
+import {removeUndefinedLocaleResources} from 'sanity'
+
+export default removeUndefinedLocaleResources({
+  /** Label for action "Copy to clipboard", tied to the "Query URL" field. Also used for accessibility purposes on button */
+  'action.copy-url-to-clipboard': 'Капіяваць у буфер абмену',
+  /** Label for stopping an ongoing listen operation */
+  'action.listen-cancel': 'Спыніць',
+  /** Label for setting up a listener */
+  'action.listen-execute': 'Слухаць',
+  /** Label for cancelling an ongoing query */
+  'action.query-cancel': 'Адмяніць',
+  /** Label for executing the query, eg doing a fetch */
+  'action.query-execute': 'Выканаць запыт',
+
+  /**
+   * Some features has a "New" label indicating that the feature was recently introduced.
+   * This defines what the text of that label is. Keep it short and sweet.
+   */
+  'label.new': 'Новае',
+
+  /** Error message for when the "Params" input are not a valid json */
+  'params.error.params-invalid-json': "Параметры не з'яўляюцца сапраўдным JSON",
+  /** Label for "Params" (parameters) editor/input */
+  'params.label': 'Параметры',
+
+  /** Label for 'Column' indicator when there is an error within the query */
+  'query.error.column': 'Слупок',
+  /** Label for 'Line' indicator when there is an error within the query */
+  'query.error.line': 'Радок',
+  /** Label for "Query" editor/input */
+  'query.label': 'Запыт',
+  /** Label for the "Query URL" field, shown after executing a query, and allows for copying */
+  'query.url': 'URL запыту',
+
+  /** Label for "End to End time" information of the fetched query */
+  'result.end-to-end-time-label': 'Час ад пачатку да канца',
+  /** Label for "Execution time" information of the fetched query */
+  'result.execution-time-label': 'Час выканання',
+  /** Label for "Result" explorer/view */
+  'result.label': 'Вынік',
+  /**
+   * "Not applicable" message for when there is no Execution time or End to End time information
+   * available for the query (eg when the query has not been executed, or errored)
+   */
+  'result.timing-not-applicable': 'не прымяняецца',
+
+  /** Label for the "API version" dropdown in settings */
+  'settings.api-version-label': 'Версія API',
+  /** Label for the "Custom API version" input in settings, shown when "other" is chosen as API version */
+  'settings.custom-api-version-label': 'Карыстацкая версія API',
+  /** Label for the "Dataset" dropdown in vision settings */
+  'settings.dataset-label': 'Dataset',
```

Comment:
@dogfrogfog For some of the "branded" product features, we suspect it is best to keep them untranslated, since other marketing materials, docs, APIs refer to them by their english name. Would that work in this locale you think?

Reply:
Yes, let's leave it as dataset

Reply:
Thanks, Ill merge now